src/components/alarm_definitions.ts

```diff
@@ -16,23 +16,42 @@
  */
 
 import appEvents from "app/core/app_events";
```

Comment:
This commit is doing a lot more than fixing a bug where the UI hangs. Really it should be a series of smaller commits.  At the very least please can we update the commit message?